src/blocks/Transaction.vue

```diff
@@ -206,12 +212,15 @@ import { RestProvider } from "zksync";
 import { warningCanceledKey } from "@/blocks/modals/TransferWarning.vue";
 import { DO_NOT_SHOW_WITHDRAW_WARNING_KEY } from "@/blocks/modals/WithdrawWarning.vue";
 
+import BuyWithRampInstant from "@/components/BuyWithRamp.vue";
```

Comment:
We don't need to import component since Nuxt 2.3.* — components are loaded async-way